ToDoApp/ToDoApp/ToDoService.swift

```diff
@@ -59,6 +60,7 @@ final class FeaturesToDo: ToDoService {
     }
     convenience init() {
         self.init(dateChecker: Calendar.current)
+        print(tasks)
```

Comment:
Is this print really needed?

Reply:
no it's just when I debug the code, but I forgot to delete it, I've fixed it.